installing/installing_openstack/installing-openstack-user-sr-iov-kuryr.adoc

```diff
@@ -14,15 +14,12 @@ Using your own infrastructure allows you to integrate your cluster with existing
 
 == Prerequisites
 
-* Review details about the
-xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
-** Verify that {product-title} {product-version} is compatible with your {rh-openstack} version by using the "Supported platforms for OpenShift clusters" section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
-
-* Verify that your network configuration does not rely on a provider network. Provider networks are not supported.
-
-* Have a {rh-openstack} account where you want to install {product-title}.
-
-* On the machine where you run the installation program, have:
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+** You verified that {product-title} {product-version} is compatible with your {rh-openstack} version by using the xref:../../architecture/architecture-installation.adoc#supported-platforms-for-openshift-clusters_architecture-installation[Supported platforms for OpenShift clusters] section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
```

Comment:
Yeah, like that!

Reply:
I've also changed this bullet point to be on the same level as the other ones.